src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -980,14 +980,18 @@ def _getPilotOptions(self, queue, **kwargs):
         else:
             self.log.info("DIRAC project will be installed by pilots")
 
-        # Pilot Logging defined?
-        pilotLogging = opsHelper.getValue("/Services/JobMonitoring/usePilotsLoggingFlag", False)
-        if pilotLogging:
-            pilotOptions.append("-z ")
+        # Pilot Logging defined? This enables the extended (possibly remote) logger.
+        # We need the URL and an optional flag which allows fine tuning on VO by VO basis:
+        pilotLogging = opsHelper.getValue("/Pilot/RemoteLogging", True)
```

Comment:
Again I would start with 

```suggestion
        pilotLogging = opsHelper.getValue("/Pilot/RemoteLogging", False)
```